### ISSUE TITLE
Cannot read properties of undefined (reading 'isPolarisWrapper') in Studio

### ISSUE THREAD

**Issue by mattdjohnson** [jneale/Xplore#75]
Hi there! In studio I see the following error:
![image](https://user-images.githubusercontent.com/3252679/198740734-df16b2db-3d04-4152-b496-d56f3237ed01.png)

![image](https://user-images.githubusercontent.com/3252679/198740804-e1b45b77-9404-4d5f-ab1f-dd8b19d43a38.png)

Some additional error checking like this should work trying to check for UI16:
```javascript
// Check for top.NOW
if(top.NOW && top.NOW.hasOwnProperty('isPolarisWrapper')){
isPolaris = top.NOW.isPolarisWrapper;
}	
```

**Comment by jneale** [comment]
Thanks! I'll add the fix in the next release.

**Comment by jneale** [comment]
Fixed in v4.13
